InteractiveHtmlBom/web/ibom.css

```diff
@@ -227,6 +227,10 @@ canvas:active {
   background-color: #313b40;
 }
 
+.bom tr.checked {
+  color: grey;
```

Comment:
This works well in dark mode but is hardly noticeable in standard theme. Maybe play with background?
If necessary create separate .dark .bom tr.checked rule.

Reply:
Done. I tried changing the background, but found it distracting, so made the standard color lighter.